.github/workflows/staticmajor.yml

```diff
@@ -0,0 +1,22 @@
+name: Staticmajor
```

Comment:
Can we add a brief description above for explaining what this workflow does?

Reply:
Done, I've made the update, thank you @julienrbrt!